settings.gradle

```diff
@@ -9,14 +9,17 @@ plugins {
     id("io.micronaut.build.shared.settings") version "6.5.7"
 }
 
+enableFeaturePreview 'TYPESAFE_PROJECT_ACCESSORS'
+
 rootProject.name = 'cassandra-parent'
 include 'cassandra'
 include 'cassandra-bom'
 include 'test-suite-graal'
 include 'tests:cassandra-jackson'
+include 'tests:cassandra-micrometer'
```

Comment:
I will prefer this to be test-suite-cassandra-micrometer

Reply:
resolved via https://github.com/micronaut-projects/micronaut-cassandra/pull/377/commits/94d8ea7dd451fab32ffd35519fab673b572c254a 

I also used the Gradle Kotlin DSL